prow/gerrit/adapter/adapter.go

```diff
@@ -50,19 +50,30 @@ const (
 )
 
 var gerritMetrics = struct {
-	processingResults     *prometheus.CounterVec
-	triggerLatency        *prometheus.HistogramVec
-	changeProcessDuration *prometheus.HistogramVec
-	changeSyncDuration    *prometheus.HistogramVec
+	processingResults           *prometheus.CounterVec
+	inrepoconfigFailures        *prometheus.CounterVec
```

Comment:
Please follow the same pattern as `processingResults` to create a SUCCESS/ERROR counter rather than just counting errors. Tracking both through the same metric via a result label lets us calculate failure ratios which are more appropriate for metrics and alerts than raw failure counts.

Reply:
Ack, will update.